File: Java/soma_vetor.java
import java.util.Locale;
import java.util.Scanner;

public class soma_vetor {
    public static void main(String[] args) {

        Locale.setDefault(Locale.US);
        Scanner input = new Scanner(System.in);

        int limite;
        double soma, media;
        soma = 0;
        media = 0;

        System.out.println("Quantos numeros voce vai digitar? ");
        limite = input.nextInt();

        double[] vet = new double[limite];

        for (int i = 0; i < limite; i++) {
            System.out.println("Digite um numero: ");
            vet[i] = input.nextDouble();
        }

        for (int i = 0; i < limite; i++) {
            soma += vet[i];
        }

        media = soma / limite;

        System.out.print("VALORES = ");
        for (int i = 0; i < limite; i++) {
            System.out.print("\t  " + String.format("%.1f", vet[i]));
        }

        System.out.println("\nSOMA = " + String.format("%.2f", soma));
        System.out.println("MEDIA = " + String.format("%.2f", media));

        input.close();
    }
}

File: Python/soma_vetor.py
'''Faça um programa que leia N números reais e armazene-os em um vetor. Em seguida: - Imprimir todos os elementos do vetor
- Mostrar na tela a soma e a média dos elementos do vetor'''

soma = float(0)
media = float(0)

quantidadeNumeros = int(input("Quantos numeros voce vai digitar? "))

vetor = [0 for x in range(quantidadeNumeros)]

for i in range(0, quantidadeNumeros):
    vetor[i] = float(input("Digite um numero: "))

print()
print("Valores: ", end="")

for i in range(0, quantidadeNumeros):
    print(f"{vetor[i]:.1f} ", end="")

print()
#CALCULANDO A SOMA DO VETOR
for i in range(quantidadeNumeros):
    soma += float(vetor[i])

#CALCULANDO A MEDIA DO VETORt
for i in range(quantidadeNumeros):
    media = float(soma / quantidadeNumeros)

print(f"Soma: {soma:.2f}")
print(f"Média: {media:.2f}")    
